modules/KIWILinuxRC.sh

```diff
@@ -1324,8 +1324,10 @@ EOF
     # install grub2 in BIOS mode
     #--------------------------------------
     if [ ! -z "$kiwi_OfwGrub" ];then
+        local prepdev=$imagePrepDevice
+        test -z $prepdev && prepdev=$(ddn $imageDiskDevice $kiwi_OfwGrub)
```

Comment:
but this code in kiwi is still broken, the ppc oemboot suse-repart has a function named

```
setupDeviceNames rootID swapID recoID bootID iorwID vgroup prepID
```

The prep id which causes imagePrepDevice to become exported is the seventh parameter.
However this parameter is never set in any of the function calls

Thus I would expect this variable to be empty all the time. actually this code should be cleaned up. imagePrepDevice is only exported in suse-preinit but never used. The only use of the variable is the code part you have changed.

Given that the variable is not useful at all I suggest we delete it from suse-preinit and in KIWILinuxRC.sh and only go with validating $kiwi_OfwGrub

Thoughts ?